bats/sql-add.bats

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bats
+load $BATS_TEST_DIRNAME/helper/common.bash
+
+setup() {
+    setup_common
+
+    dolt sql <<SQL
+CREATE TABLE test (
+    pk int primary key
+);
+CREATE TABLE test2 (
+    pk int primary key
+);
+INSERT INTO test VALUES (0),(1),(2);
+SQL
+    dolt sql <<SQL
+DELETE FROM test WHERE pk = 0;
+INSERT INTO test VALUES (3);
+SQL
+}
+
+teardown() {
+    teardown_common
+}
+
+@test "DOLT_ADD all flag works" {
+    run dolt sql -q "SELECT DOLT_ADD('-A')"
+    run dolt sql -q "SELECT DOLT_COMMIT('-m', 'Commit1')"
+
+    # Check that everything was added
+    run dolt diff
+    [ "$status" -eq 0 ]
+    [ "$output" = "" ]
+
+    run dolt log
+    [ $status -eq 0 ]
+    [[ "$output" =~ "Commit1" ]] || false
+    regex='Bats Tests <bats@email.fake>'
+    [[ "$output" =~ "$regex" ]] || false
+}
+
+@test "DOLT_ADD can take in one table" {
+    run dolt sql -q "SELECT DOLT_ADD('test')"
+    run dolt sql -q "SELECT DOLT_COMMIT('-m', 'Commit1')"
+
+    # Check that just test was added and not test2.
+    run dolt status
+    [ "$status" -eq 0 ]
+    regex='test2'
+    [[ "$output" =~ "$regex" ]] || false
+
+    run dolt log
+    [ $status -eq 0 ]
+    [[ "$output" =~ "Commit1" ]] || false
+    regex='Bats Tests <bats@email.fake>'
+    [[ "$output" =~ "$regex" ]] || false
+}
+
+
+@test "DOLT_ADD can take in multiple tables" {
+    run dolt sql -q "SELECT DOLT_ADD('test', 'test2')"
+    run dolt sql -q "SELECT DOLT_COMMIT('-m', 'Commit1')"
+
+    # Check that both test and test2 are added.
+    run dolt diff
+    [ "$status" -eq 0 ]
+    [ "$output" = "" ]
+
+    run dolt log
+    [ $status -eq 0 ]
+    [[ "$output" =~ "Commit1" ]] || false
+    regex='Bats Tests <bats@email.fake>'
+    [[ "$output" =~ "$regex" ]] || false
+}
```

Comment:
You should have a test that makes the handling of README.md files explicit. I think we decided that it doesn't make sense to deal with the local file system from SQL? So have a test that writes a README.md file, does SELECT  dolt_add(".") and commits, and confirms that README.md still hasn't been committed.

Reply:
Right now dolt add works with docs. Should I remove that behavior?

Reply:
I think it's fine as is, we can change later.